src/WebJobs.Script.WebHost/web.config

```diff
@@ -9,6 +9,11 @@
       <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModule" resourceType="Unspecified"/>
     </handlers>
     <aspNetCore processPath="%LAUNCHER_PATH%" arguments="%LAUNCHER_ARGS%" stdoutLogEnabled="false" />
+    <security>
+      <requestFiltering>
+        <requestLimits maxQueryString="4069" maxUrl="8192"/>
```

Comment:
Shouldn't we match these settings with [V1](https://github.com/Azure/azure-functions-host/blob/v1.x/src/WebJobs.Script.WebHost/Web.config#L53) ?
```
 <requestFiltering allowDoubleEscaping="true">
      <requestLimits maxAllowedContentLength="104857600" maxUrl="8192" maxQueryString="4096"/>
  </requestFiltering>
```

Reply:
Wasn't sure if we wanted to limit payload size. This would limit it to 100 MB.

Reply:
Looks like it was set to match logic app payload size ( https://github.com/Azure/azure-functions-host/commit/823f7c153e8444fc718bc70106a513dadb1e7dca). The default is 30 MB, changing it to 100MB